client/src/components/LinkField/LinkField.js

```diff
@@ -19,36 +22,63 @@ const section = 'SilverStripe\\LinkField\\Controllers\\LinkFieldController';
  * onChange - callback function passed from JsonField - used to update the underlying <input> form field
  * types - injected by the GraphQL query
  * actions - object of redux actions
+ * isMulti - whether this field handles multiple links or not
  */
-const LinkField = ({ value, onChange, types, actions }) => {
-  const linkID = value;
-  const [typeKey, setTypeKey] = useState('');
+const LinkField = ({ value = null, onChange, types, actions, isMulti = false  }) => {
   const [data, setData] = useState({});
-  const [editing, setEditing] = useState(false);
+  const [editingID, setEditingID] = useState(null);
+
+  // Ensure we have a valid array
+  let linkIDs = value;
+  if (!Array.isArray(linkIDs)) {
+    if (typeof linkIDs === 'number') {
+      linkIDs = [linkIDs];
+    }
+    if (!linkIDs) {
+      linkIDs = [];
+    }
+  }
+
+  // Read data from endpoint and update component state
+  // This happens any time a link is added or removed and triggers a re-render
+  useEffect(() => {
+    if (!editingID && linkIDs.length > 0) {
+      const query = [];
+      for (const linkID of linkIDs) {
+        query.push(`itemID[]=${linkID}`);
+      }
+      const endpoint = `${Config.getSection(section).form.linkForm.dataUrl}/?${query.join('&')}`;
```

Comment:
```suggestion
      const endpoint = `${Config.getSection(section).form.linkForm.dataUrl}?${query.join('&')}`;
```
Don't need the trailing slash since there's no params being used on the endpoint

Changes 
/admin/linkfield/data/?itemID[]=6&itemID[]=10&itemID[]=11&itemID[]=12
to
/admin/linkfield/data?itemID[]=6&itemID[]=10&itemID[]=11&itemID[]=12

Reply:
Removed to avoid ping pong, though it doesn't affect anything one way or another.